.travis.yml

```diff
@@ -1,36 +1,40 @@
-language: python
 dist: xenial
+
+language: python
+
 jobs:
   include:
-    - { python: "3.6",   env: "TOXENV=py36" }
-    - { python: "3.7",   env: "TOXENV=py37" }
     - { python: "3.8",   env: "TOXENV=py38" }
+    - { python: "3.7",   env: "TOXENV=py37" }
+    - { python: "3.6",   env: "TOXENV=py36" }
 
-    - stage: PyPI Release
+    - stage: PyPI release
       if: tag IS present
-      python: "3.7"
+      python: "3.8"
       env: []
       before_install: skip
       install: skip
       script: skip
       deploy:
         provider: pypi
         user: itajaja
-        on:
```

Comment:
double checking this is intentional

Reply:
first commit was messed up; meant to add https://github.com/4Catalyzer/fourmat/pull/16/commits/f673492a794ad181dbf3866c91123713da615d21#diff-354f30a63fb0907d4ad57269548329e3R12

Reply:
ah shoot this is necessary